tests/test_loads.py

```diff
@@ -50,76 +50,71 @@ def run_gdb_with_script(binary='', core='', pybefore=None, pyafter=None):
     return output
 
 
-HELLO = (
-    'pwndbg: loaded ### commands. Type pwndbg [filter] for a list.\n'
-    'pwndbg: created $rebase, $ida gdb functions (can be used with print/break)\n'
-)
+HELLO = [
+    'pwndbg: loaded ### commands. Type pwndbg [filter] for a list.',
+    'pwndbg: created $rebase, $ida gdb functions (can be used with print/break)'
+]
```

Comment:
To solve these kinds of tests, I did conversions from tuples and strings to lists of strings.